libvmi/private.h

```diff
@@ -183,12 +183,11 @@ typedef struct _windows_unicode_string32 {
     win32_unicode_string_t;
 
 /** Windows' UNICODE_STRING structure (x64) */
-    typedef struct _windows_unicode_string64 {
-        uint16_t length;
-        uint16_t maximum_length;
-        uint64_t pBuffer;   // pointer to string contents
-    } __attribute__ ((packed))
-    win64_unicode_string_t;
+typedef struct _windows_unicode_string64 {
+    uint16_t length;
+    uint16_t maximum_length;
```

Comment:
We should have an explicit 32-bit padding field here so that we don't rely on the compiler replicating the padding of the microsoft compiler exactly.